src/components/Card/Card.js

```diff
@@ -34,6 +34,12 @@ const CardList = styled.div`
   flex-direction: column;
   width: 223px;
   margin: 0 8px;
+  transform: scale(0.9);
+  transition: all 0.3s;
+
+  :hover {
+    transform: scale(1);
+  }
```

Comment:
자식 element가 아닌 해당 element에 hover했을 때도 적용하려면 부모 가상 선택자를 붙여주세요!
```suggestion
  &:hover {
    transform: scale(1);
  }
```